python/basic/advanced_img_io.ipynb

```diff
@@ -4,6 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Download the whole repository with command:\n",
```

Comment:
I think the simplest way for user to manage notebooks is to download the whole repository, instead individual file.